vignettes/design-principles.Rmd

```diff
@@ -29,6 +29,34 @@ This package should work with plots provided by:
 - ggplot2
 - ggraph
 
+## Design decisions
+
+### Alignment with Epiverse-TRACE web theme
+
+Epiverse-TRACE already has a clear visual identity for its web resources, via the [pkgdown custom theme](https://epiverse-trace.github.io/epiversetheme), the [varnish fork to customize the look of online training materials](https://epiverse-trace.github.io/varnish), and the [custom theme for the quarto website](https://github.com/epiverse-trace/epiverse-trace.github.io/blob/0d70e09531c422648e0a1d46f7ca64c45def78a2/custom.scss).
+
+The custom ggplot2 theme and scale provided here should roughly align with this existing visual identity.
+
+In practice, this means:
+
+- The default font should be [`DM Sans`](https://fonts.google.com/specimen/DM+Sans), or [`Roboto Mono`](https://fonts.google.com/specimen/Roboto+Mono) for code.
```

Comment:
Could you expand on how this would be achieved? Is the theme package intended to bundle these fonts?

Reply:
Maybe I'm being too narrow-minded, but shouldn't we include the corporate typography? (Clash Display/ Erode/ Open Sans).

Reply:
> Could you expand on how this would be achieved? Is the theme package intended to bundle these fonts?

Let's discuss about implementation in a separate issue.

> Maybe I'm being too narrow-minded, but shouldn't we include the corporate typography? (Clash Display/ Erode/ Open Sans).

DM Sans and Roboto Mono are already part of the brand on the website theme, which was created with David's, data.org designer, guidance.

My understanding is that our license for Clash Display doesn't allow us to make it available to external users.